test/src/sdk-harness/test_projects/block/src/main.sw

```diff
@@ -19,4 +19,12 @@ impl BlockTest for Contract {
     fn get_block_and_timestamp() -> (u64, u64) {
         (height(), timestamp())
     }
+
+    fn get_block_header_hash(h: u64) -> b256 {
```

Comment:
I'd consider whether it would be better to return the Result instead and test for the variant and value